.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -40,3 +40,10 @@ Also make sure to read the guidelines found at
 - If this PR adds a new module
 
   - [ ] Added myself as module maintainer. See [example](https://github.com/nix-community/home-manager/blob/068ff76a10e95820f886ac46957edcff4e44621d/modules/programs/lesspipe.nix#L6).
+
+#### Maintainer CC
```

Comment:
Why h4 instead of h3 like the other headers?

Reply:
It's sort of part of the checklist and i don't want it to take up too much space